app/services/announcements.ts

```diff
@@ -47,6 +51,13 @@ export class AnnouncementsService extends StatefulService<IAnnouncementsInfo> {
     const req = this.formRequest(endpoint);
     try {
       const newState = await fetch(req).then((rawResp) => rawResp.json());
+      const queryString = newState.link.split('?')[1];
```

Comment:
Can you maybe add a comment showing what the expected link should look like with params?  I'm assuming you're parsing params out of the link field mainly so we don't have to make any backend changes?